gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/GfxdDDLMessage.java

```diff
@@ -274,8 +274,10 @@ protected void processMessage(final DistributionManager dm)
       }
       return;
     }
-    SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_DDLREPLAY, this.toString()
-        + " Starting execution");
+    if (!Misc.isSnappyHiveMetaTable(ddl.getCurrentSchema())) {
+      SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_DDLREPLAY, this.toString()
+          + " Starting execution");
+    }
```

Comment:
Not clear why these additional checks are required even after it has been altered to WARNING both in DDL replay and externalCatalog initialization.